test/asn1choice.spec.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as assert from "assert";
+import * as asn1js from "../src";
+import { SchemaContext } from "../src";
+import { ETagClass } from "../src/TypeStore";
+
+/**
+ * Converts an array buffer to hex notation
+ *
+ * @param buffer - the buffer to convert
+ * @returns the buffer in hex string notation
+ */
+function buf2hex(buffer: ArrayBuffer): string {
```

Comment:
I see you implemented some helping functions like `buf2hex`, `hex2buf` and use their copies in each test file. `pvtsutils` has `Convert` static class which implements this functionality, please use it